src/components/map/WorldMap.tsx

```diff
@@ -0,0 +1,339 @@
+  );
+};
+
+// TODO:
+// - Add document type selector
+
+export default function MapChart() {
+  const configQuery = useConfig();
+  const geographiesQuery = useGeographies();
+  const { data: { countries: configContries = [] } = {} } = configQuery;
+  const { data: mapData = [], status: mapDataStatus } = geographiesQuery;
+  const geographyInfoTooltipRef = useRef<TooltipRefProps>(null);
+  const mapRef = useRef(null);
+  const [activeGeography, setActiveGeography] = useState("");
+  const [mapCenter, setMapCenter] = useState<TPoint>([0, 0]);
+  const [mapZoom, setMapZoom] = useState(1);
+  const [showUnifiedEU, setShowUnifiedEU] = useState(false);
+  const [selectedDocType, setSelectedDocType] = useState<"lawsPolicies" | "unfccc">("lawsPolicies");
+
+  // Combine the data from the coordinates and the map data from the API into a unified object
+  const geographiesWithCoords: TGeographiesWithCoords = useMemo(() => {
+    // Calculate size of marker
+    const maxLawsPolicies = Math.max(...mapData.map((g) => g.family_counts?.EXECUTIVE || 0 + g.family_counts?.LEGISLATIVE || 0));
+    // Only take UNFCCC counts for countries that are not XAA or XAB (international, no geography)
+    const maxUNFCCC = Math.max(...mapData.map((g) => (["XAA", "XAB"].includes(g.iso_code) ? 0 : g.family_counts?.UNFCCC || 0)));
+
+    return configContries.reduce((acc, country) => {
+      const geoStats = mapData.find((geo) => geo.slug === country.slug);
+      acc[country.value] = {
+        ...country,
+        coords: GEO_CENTER_POINTS[country.value],
+        familyCounts: geoStats?.family_counts,
+        markers: {
+          lawsPolicies: Math.max(
+            minMarkerSize,
+            ((geoStats?.family_counts?.EXECUTIVE || 0 + geoStats?.family_counts?.LEGISLATIVE || 0) / maxLawsPolicies) * maxMarkerSize
+          ),
+          unfccc: Math.max(minMarkerSize, ((geoStats?.family_counts?.UNFCCC || 0) / maxUNFCCC) * maxMarkerSize),
+        },
+      };
+      return acc;
+    }, {});
+  }, [configContries, mapData]);
+
+  const handleGeoClick = (e: React.MouseEvent<SVGPathElement>, geo: TSvgGeo) => {
+    setActiveGeography(geo.properties.name);
+    const geography = Object.values(geographiesWithCoords).find((g) => g.display_value === geo.properties.name);
+    openToolTip([e.clientX, e.clientY], geography?.display_value ?? geo.properties.name);
+  };
+
+  const handleMarkerClick = (e: React.MouseEvent<SVGPathElement>, countryCode: string) => {
+    const geography = geographiesWithCoords[countryCode];
+    openToolTip([e.clientX, e.clientY], geography?.display_value ?? "");
+  };
+
+  const handleGeoHover = (e: React.MouseEvent<SVGPathElement>, hoveredGeo: string) => {
+    setActiveGeography("");
+    openToolTip([e.clientX, e.clientY], hoveredGeo);
+  };
+
+  const handleGeographySelected = (selectedCountry: TGeographyWithCoords) => {
+    setActiveGeography(selectedCountry.display_value);
+    setMapCenter(selectedCountry.coords);
+    setMapZoom(5);
```

Comment:
Be interesting to know how the map behaves with browser zoom too - guessing "5" is a fixed level?

Reply:
The map stays relatively consistent in regards to its rendering (I presume due to it being an SVG) whereas the UI around it does obviously increase as you zoom. I hadn't realised this would be the case.

The zoom here refers to the `react-simple-maps` configuration of zoom on a scale of 1-9. 1 being incredibly zoomed in to the point of uselessness (particular given we are not using a hyper-detailed map) and 9 being the entire world.